docs/assets/0123-import-scripts/README.md

```diff
@@ -0,0 +1,142 @@
+shortcode, and `import` by your project's shortname. This is especially recommended for big projects where you spend 
+weeks/months on, when you might want to have a backup, or when you want to invite colleagues for collaboration or a code
+review.
+
+
+## Choose a Python interpreter
+Open `import-script.py`. You can now choose a Python interpreter by clicking on the Version number on the bottom right.
+You can either work with the global (system-wide) Python, or you can create a 
+[virtual environment](https://python.land/virtual-environments) for your project. If you don't know which one to choose, 
+take the one installed via Homebrew, which is located in `/usr/local/Cellar`. Probably you already have a symlink 
+(`/usr/local/bin/python3` or `/usr/bin/python3`) that redirects to `/usr/local/Cellar`. The only thing that you 
+shouldn't do is selecting a virtual environment of another project.
+
+![python interpreter](assets/python-interpreter.png)
+
+
+## The benefits of the debugging mode
+To start the debugging process, switch to the "Run and Debug" tab.
+
+1. set a break point
+2. click "Run and Debug"
+3. choose "Debug the currently active Python file"
+4. The control bar appears, and debugging starts.
+
+![configure debugging](assets/configure-debugging.png)
+
+Code execution will interrupt at your break point, that means, before the line of the break point is executed. Use 
+this opportunity to inspect what has been done until now in the "Variables" area on the left, where the current state of
+the program is shown.  
+
+If one of the dependencies is not installed, install it with `pip install package` in the Terminal of Visual Studio 
+Code.
+
+![pause at breakpoint](assets/pause-at-breakpoint.png)
+
+If you want to experiment with different scenarios how to proceed, go to the "Debug Console" where you can execute 
+code. For example, let's inspect the Pandas Dataframe by typing `main_df.info`.  
+You see that there are some empty rows at the end which don't contain useful data. The next two lines of code will 
+eliminate them. Click on "Step Over" two times, or set a new break point two lines further down and click on "Continue".
+Now, type again `main_df.info` in the Debug Console. You will see that the empty rows are gone.  
+
+![inspect dataframe](assets/inspect-dataframe.png)
+
+
+You see that the debugging mode is a useful tool to understand code and to inspect it for correctness. 
+
+| **Tip**                                                                                                   |
+|-----------------------------------------------------------------------------------------------------------|
+| **Make regular use of the debugging mode to check if your code really does what you think it should do!** | 
+
+
+## The benefits of version control
+One of the big benefits of version control is the diff viewer. Visual Studio highlights the changes you have introduced 
```

Comment:
Visual Studio is a different product from Visual Studio Code. Those two should not be mixed up